bundles/org.openhab.binding.irobot/src/main/feature/feature.xml

```diff
@@ -18,7 +18,6 @@
 
 	<feature name="openhab-binding-irobot" description="iRobot Binding" version="${project.version}">
 		<feature>openhab-runtime-base</feature>
-		<feature dependency="true">openhab.tp-hivemqclient</feature>
```

Comment:
You need to add `<feature>openhab-transport-mqtt</feature>`. This is why the build fails.

Reply:
Thank you for the hint, fixed